### PR TITLE
feat: prompter over/under timer

### PR DESCRIPTION
**Pull request by jstarpl**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

This is a new feature

* **What is the current behavior?** (You can also link to an open issue here)

There is no over/under information in the prompter

* **What is the new behavior (if this is a feature change)?**

An over/under timer is shown in the prompter screen:

obraz

* **Other information**:

**Status**

- [x] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [x] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/client/ui/Prompter/PrompterView.tsx [hunk 1]
```diff
@@ -113,14 +117,21 @@ export class PrompterViewInner extends MeteorReactComponent<Translated<IProps &
...
+				{
+					fields: {
+						_id: 1,
+					},
+				}
+			)
+			if (playlist?._id) {
 				this.subscribe(PubSub.rundowns, {
-					playlistId: playlistId,
+					playlistId: playlist?._id,
```
Comment (nytamin) [inline meteor/client/ui/Prompter/PrompterView.tsx:134]: minor note: the optional chaining `?.` is unnecessary here.

File: meteor/lib/api/prompter.ts [hunk 2]
```diff
@@ -75,14 +75,14 @@ export namespace PrompterAPI {
 				parts: [],
 			}
 
-			_.each(parts, (part) => {
+			parts.forEach((part) => {
```
Comment (nytamin) [inline meteor/lib/api/prompter.ts:78]: changing this inner loop to a for-loop might have slight improvements in performance.

File: meteor/lib/api/prompter.ts [hunk 3]
```diff
@@ -75,14 +75,14 @@ export namespace PrompterAPI {
...
 				const partData: PrompterDataPart = {
 					id: part._id,
 					title: part.title,
 					pieces: [],
 				}
 
-				_.each(part.getAllPieces(), (piece) => {
+				part.getAllPieces().forEach((piece) => {
```
Comment (nytamin) [inline meteor/lib/api/prompter.ts:85]: changing this inner loop to a for-loop might have slight improvements in performance.

**Review by nytamin — APPROVED** [review]
Looks good! Just a few minor notes.

File: meteor/client/ui/Prompter/PrompterView.tsx (hunk 1 above)
Comment (jstarpl) [reply meteor/client/ui/Prompter/PrompterView.tsx]: Removed the optional chaining.

File: meteor/lib/api/prompter.ts (hunk 2 above)
Comment (jstarpl) [reply meteor/lib/api/prompter.ts]: Switched to a `for ... of` loop.

File: meteor/lib/api/prompter.ts (hunk 3 above)
Comment (jstarpl) [reply meteor/lib/api/prompter.ts]: Switched to a `for ... of` loop.

**Pull request merged**